### PR TITLE
Merge Police - update with last Gendarmerie file

### PR DESCRIPTION
**Pull request by nlehuby**
* new file url
* use a conflation key
* update a few tags in prevision of https://wiki.openstreetmap.org/wiki/FR:Project_of_the_month/Gendarmerie_nationale

### CODE REVIEW

File: analysers/analyser_merge_police_FR.py [hunk 1]
```diff
@@ -26,22 +26,31 @@
...
+            CSV(Source(attribution = u"data.gouv.fr:Ministère de l'Intérieur", millesime = "10/2018",
+                    fileUrl = "https://www.data.gouv.fr/fr/datasets/r/d6a43ef2-d302-4456-90e9-ff2c47cac562"),
                 separator = ";"),
             Load("geocodage_x_GPS", "geocodage_y_GPS"),
             Mapping(
                 select = Select(
                     types = ["nodes", "ways"],
                     tags = {"amenity": "police"}),
-                conflationDistance = 1000,
+                conflationDistance = 500,
```
Comment (frodrigo) [inline analysers/analyser_merge_police_FR.py]: It's required to lower the distance ?
There is existing Gendarmerie closer than 1000m ?

File: analysers/analyser_merge_police_FR.py [hunk 2]
```diff
@@ -26,22 +26,31 @@
...
+                osmRef = "ref:data.gouv.fr",
                 generate = Generate(
                     static1 = {
                         "amenity": "police",
-                        "operator": "Gendarmerie Nationale"},
+                        "name": "Gendarmerie nationale",
+                        "operator": "Gendarmerie nationale"},
                     static2 = {"source": self.source},
-                    mapping2 = {"phone": "telephone"},
+                    mapping1 = {"ref:data.gouv.fr": "identifiant_public_unite"},
```
Comment (frodrigo) [inline analysers/analyser_merge_police_FR.py]: It's a stable ref ?

Comment (nlehuby) [reply analysers/analyser_merge_police_FR.py]: We can hope so, its purpose is to ease the update for the data user: https://twitter.com/ericfreyss/status/1045750462690070530

Comment (frodrigo) [reply analysers/analyser_merge_police_FR.py]: Ok. But not better ref name? this one is not specific for this data source.

File: analysers/analyser_merge_police_FR.py (hunk 1 above)
Comment (nlehuby) [reply analysers/analyser_merge_police_FR.py]: the geoloc does not look that bad, and I would like to reduce the risk to conflate with police office that are not Gendarmerie.
Do you prefer I change it back to 1 km ?

File: analysers/analyser_merge_police_FR.py (hunk 2 above)
Comment (nlehuby) [reply analysers/analyser_merge_police_FR.py]: What do you suggest ? `ref:FR:GendarmerieNationale` ?

Comment (frodrigo) [reply analysers/analyser_merge_police_FR.py]: yes

Comment (nlehuby) [reply analysers/analyser_merge_police_FR.py]: ok

**Comment by frodrigo** [comment]
http://osmose.openstreetmap.fr/fr/errors/?item=8190,7250,8191,8192&useDevItem=true

**Comment by frodrigo** [comment]
I change the wording for homogeneity with other.
But there is some kind of bug, item=8190 have no content.

**Comment by frodrigo** [comment]
I finally found the 8190 bug, it was on front of the keyboard.
The good URL is:
http://osmose.openstreetmap.fr/fr/errors/?item=8190,7250,8191,8192&useDevItem=all
http://osmose.openstreetmap.fr/fr/errors/graph.png?source=6388&item=8190&class=1

**Comment by frodrigo** [comment]
In fact we should not set the missing_osm item=7250 because it reports on all amenity=police the missing ref:FR:GendarmerieNationale. But in France amenity=police can be "Gendarmerie" or "Police".

**Pull request merged**